package.json

```diff
@@ -6,7 +6,7 @@
   "homepage": "https://github.com/mozilla/glam#readme",
   "repository": {
     "type": "git",
-    "url": "git@github.com:mozilla/glam.git"
+    "url": "git+ssh://git@github.com/mozilla/glam.git"
```

Comment:
Curious why this changed?

Reply:
not sure. Maybe when I ran npm it fetched something from my git config?
In any case I'm reverting all changes in this file